doc/classes/ProjectSettings.xml

```diff
@@ -70,15 +70,19 @@
 		<method name="get_setting" qualifiers="const">
 			<return type="Variant" />
 			<param index="0" name="name" type="String" />
+			<param index="1" name="default_value" type="Variant" default="null" />
 			<description>
 				Returns the value of a setting.
+				If a default value is set, it is returned when the setting does not exist.
 				[b]Example:[/b]
 				[codeblocks]
 				[gdscript]
 				print(ProjectSettings.get_setting("application/config/name"))
+				print(ProjectSettings.get_setting("is_my_custom_setting_set", false))
```

Comment:
```suggestion
				print(ProjectSettings.get_setting("application/config/custom_description", "No description speficied."))
```

I think the example name may be misleading a bit. Also using a boolean here may be confusing, because we want to show that the property can have a fallback value, but a boolean may read like an indicator if the property exists or not.